src/hooks/useDebounceRules.ts

```diff
@@ -45,21 +46,28 @@ export const useDebounceRules = (
 
   // Update errors according to rules
   const onCheckRules = (id: string, value: string, password?: string) => {
-    const rule = authRules.find((r) => r.id === id);
-    let validationResult = '';
-    if (rule) {
-      switch (id) {
-        case 'passwordConfirmation':
-          validationResult = rule.validate(value, password);
-          break;
-        default:
-          validationResult = rule.validate(value);
-          break;
+    if (caller === 'auth') {
+      const rule = authRules.find((r) => r.id === id);
+      let validationResult = '';
+      if (rule) {
+        switch (id) {
+          case 'passwordConfirmation':
+            validationResult = rule.validate(value, password);
```

Comment:
Do we have to create a new if every time we have a new form? Due to time constraints its fine, but I don't think its very scalable

Reply:
It is not scalable at all, but we only have one more file with forms up to this moment, so I don't think it will be the end of the world

Reply:
It is not scalable at all, but we only have one more file with forms up to this moment, so I don't think it will be the end of the world